.pluginsync.yml

```diff
@@ -0,0 +1,18 @@
+# File managed by pluginsync
+pluginsync_config: '0.1.8'
+managed_files:
+- .github
+- .github/ISSUE_TEMPLATE.md
+- .github/PULL_REQUEST_TEMPLATE.md
+- .gitignore
+- .pluginsync.yml
+- .travis.yml
+- CONTRIBUTING.md
+- LICENSE
+- Makefile
+- scripts
+- scripts/build.sh
+- scripts/common.sh
+- scripts/deps.sh
+- scripts/pre_deploy.sh
+- scripts/test.sh
```

Comment:
@nanliu Despite the fact that `Makefile`, `scripts/build.sh`, `scripts/test.sh` are unmanaged and I see the following output from snap-pluginsync:
```
Not managing Makefile in snap-plugin-publisher-heka
Not managing scripts/build.sh in snap-plugin-publisher-heka
Not managing scripts/test.sh in snap-plugin-publisher-heka
```
these files are mentioned in `.pluginsync.yml`. It seems to be an issue in snap-pluginsync. What do you thing?

The full log from snap-pluginsync is available [here](https://gist.github.com/katarzyna-z/f4c3b7a9dec3531a8679accabd66b177),